src/server/handlers/metrics.ts

```diff
@@ -26,7 +26,11 @@
 import Metrics from '@mojaloop/central-services-metrics'
 import { Request, ResponseToolkit, ResponseObject } from '@hapi/hapi'
 
-export function get (_: Request, h: ResponseToolkit): ResponseObject {
+function get (_context: any, _: Request, h: ResponseToolkit): ResponseObject {
```

Comment:
use here Context interface you defined in`~/server/plugins`

Reply:
yes, I think I didn't save in VSCode, I fixed it now. thanks for catching this